### PR TITLE
Adds script to auto start importer

### PR DESCRIPTION
**Pull request by K8Sewell**
# Summary
Adds a rake task to check for running import jobs and if none are running will trigger another import job to be run.

# Related Ticket
[i124](https://github.com/scientist-softserv/ams/issues/124)

# Co-Authored By:
* JP - @jpengst
* Amanda - @akfosterdev

### CODE REVIEW

File: lib/tasks/ams/trigger_importer.rake [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require 'ams'
```
Comment (bkiahstroud) [inline lib/tasks/ams/trigger_importer.rake]: Why is this line here?

**Review by bkiahstroud — COMMENTED** [review]
I'd love to see some logging, even if it's super bare bones. Would y'all mind if I just implement this myself? I can do it pretty quick

File: lib/tasks/ams/trigger_importer.rake (hunk 1 above)
Comment (K8Sewell) [reply lib/tasks/ams/trigger_importer.rake]: It was in the other rake tasks so we assumed it was necessary but can pull if it's not.

**Comment by K8Sewell** [comment]
> I'd love to see some logging, even if it's super bare bones. Would y'all mind if I just implement this myself? I can do it pretty quick

Yes, please feel welcome to add.  Sorry that logging slipped our minds when we were mobbing.  Just lmk if it's easier or faster for me to add logging for you.

**Review by bkiahstroud — APPROVED** [review]
